src/lib/drivers/rangefinder/PX4Rangefinder.cpp

```diff
@@ -66,10 +68,10 @@ void PX4Rangefinder::set_orientation(const uint8_t device_orientation)
 	_distance_sensor_pub.get().orientation = device_orientation;
 }
 
-void PX4Rangefinder::update(const hrt_abstime &timestamp_sample, const float distance, const int8_t quality)
+
+void PX4Rangefinder::update(const hrt_abstime &timestamp_sample, const float distance, const int8_t quality, const float (&quat)[4])
```

Comment:
Oh man I'm not sure I like `(&quat)[4]`, I don't think I've ever seen a reference to a pointer. I'd probably change it to 

```suggestion
void PX4Rangefinder::update(const hrt_abstime &timestamp_sample, const float distance, const int8_t quality, const float *q, int q_len = 4)
```
Since C arrays don't have any bounds checking anyway, the implementation has to assume length 4 regardless of what is passed.